qldbdriver/result_test.go

```diff
@@ -47,6 +47,19 @@ func TestResult(t *testing.T) {
 		logger:       nil,
 	}
 
+	mockReadIOs := int64(1)
```

Comment:
This isn't a mock so remove the prefix "mock". There are several cases like this.

Reply:
Removed `mock` prefixes.